### ISSUE TITLE
The training list and the values of some parameters

### ISSUE THREAD

**Issue by zheangh** [RL4M/PCRLv2#4]
Thank you for the excellent work and for releasing it.
I wonder how the train txt is generated for LUNA, because in the code you already divide the train/val set by different "subset".
In addition, I notice some hyperparameters are different between the paper and the code (e.g., batch size for 2D, weight decay), and the "patience" parameter in finetuning code is missing. Could you confirm their values? Thanks!

**Comment by liaochuanlin** [comment]
@zheangh 我也发现了这个问题请问你解决了吗？

**Comment by Fym68** [comment]
How did you solve the problem with the patience parameter? I also have this problem.
